README.md

```diff
@@ -16,8 +16,10 @@ Use this in favour of the [Rift wiki's](https://github.com/DimensionalDevelopmen
 ```groovy
 buildscript {
 	repositories {
+		mavenCentral()
 		maven { url 'https://www.jitpack.io' }
 		maven { url 'https://files.minecraftforge.net/maven' }
+		maven { url 'https://www.dimdev.org/maven/' }
```

Comment:
Is the DimDev maven actually needed for anything as it stands? If you're using Mixins I know it is but would've thought that `mavenCentral` would cover everything otherwise?

Reply:
Looking into it further where the dependencies are all coming from, Rift implicitly adds Mixin as a dependency which explains the DimDev maven for the repos block. I still don't see why this one is necessary though as it shouldn't need this unless MixinGradle demands Mixin directly too.